spec/javascripts/Stats/top_apps_help_text.spec.js

```diff
@@ -19,20 +19,20 @@ describe('StatsTopAppsHelpText', () => {
     let weekPeriod = { _topAppsSelectionPeriod: { since: '2017-01-02T00:00:00+00:00', until: '2017-01-08T23:59:59+00:00', name: 'week' } }
 
     helpText.update(weekPeriod)
-    expect($('.Stats-message--notice')).toContainText('Top Applications are determined from usage data between midnight 01/01/2017 and midnight 01/08/2017')
+    expect($('.Stats-message--notice').text()).toBe('Top Applications are determined from usage data between midnight 01/01/2017 and midnight 01/08/2017')
```

Comment:
Same as in a previous comment, I think we could take this opportunity to use querySelector instead of "abusing" $.

Reply:
Agree with removing jquery, but I was planning to do it in a following PR.
Here: just remove karma/jasmine
Next: improvements, includes removing jquery